pytest_doctestplus/plugin.py

```diff
@@ -438,7 +464,14 @@ def pytest_ignore_collect(self, path, config):
             dirpath = Path(path).parent
         else:
             dirpath = path.dirpath()
-        collect_ignore = config._getconftest_pathlist("collect_ignore", path=dirpath)
+
+        try:
+            collect_ignore = config._getconftest_pathlist("collect_ignore", path=dirpath)
+        except TypeError:
+            # Pytest 7.0+
```

Comment:
Why not doing version check here?

Reply:
I forgot about it ;). And this is a bit of a mess, some changes are in 6.3 which is not released yet, and some will be in 7.0. And dev is currently 6.3.devXXX though it includes the 7.0 changes apparently.
So I added PYTEST_GT_6_3 after but I'm still not sure if that's a good idea.

Reply:
I mean try/except is easy to do, distinguishing 6.3 and 7.0 is currently not obvious.

Reply:
Okay, maybe we can merge as is but open issue to revisit in the future because you don't want the stable pytest to keep hitting the `except` block.